.appveyor.yml

```diff
@@ -12,14 +13,15 @@ install:
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
   - if NOT "%TARGET%" == "x86_64-pc-windows-msvc" rustup target add %TARGET%
   - if defined MSYS_BITS set PATH=C:\msys64\mingw%MSYS_BITS%\bin;C:\msys64\usr\bin;%PATH%
+  - if "%TARGET%" == "wasm32-unknown-unknown" cargo install wasm-pack
```

Comment:
Oh also I think it's fine to skip this on AppVeyor, if it works on Unix I think it's safe to say it works everywhere

Reply:
Done!